tests/scaffoldings/symlinks/Dockerfile

```
@@ -0,0 +1,2 @@
+ARG ODOO_VERSION
+FROM registry.wt-io-it.at/local:ci-local-${ODOO_VERSION}-onbuild
```

Can you change this to:
```suggestion
FROM tecnativa/doodba:${ODOO_VERSION}-onbuild
```
which is what we use on the rest of the tests? 
Or is there a need to use this specific image in your test?

---

sorry, yes, will fix that, this is an artifact of my local tests